supervisor/api/host.py

```diff
@@ -43,6 +44,7 @@ async def info(self, request):
             ATTR_DISK_FREE: self.sys_host.info.free_space,
             ATTR_DISK_TOTAL: self.sys_host.info.total_space,
             ATTR_DISK_USED: self.sys_host.info.used_space,
+            ATTR_SSD_LIFE_TIME: self.sys_host.info.ssd_life_time,
```

Comment:
let's call it `ATTR_DISK_LIFE_TIME` which is aligned within `ATTR_DISK_USED` as the first start. We can later expose more and different data over the /os/ endpoint

Reply:
Suggestion (future proof) `ATTR_STORAGE_LIFE_TIME`

Reply:
It's more all other on that endpoint is called with DISK and means `/data`

Reply:
```suggestion
            ATTR_DISK_LIFE_TIME: self.sys_host.info.disk_life_time,
```